Directory.Build.targets

```diff
@@ -0,0 +1,31 @@
+<Project>
+    <Target
+            Name="PublishSamples"
```

Comment:
putting this target in every project (via D.B.targets) means that calling the target at the solution-level Just Works (thanks to solution file's traversal targets mechanism).